src/main/resources/report.csv

```diff
@@ -0,0 +1 @@
+fruit,quantity
```

Comment:
is it the correct report? How does your solution work?

Reply:
Not resolved, this file represents wrong solution, please check task requirements to see how correct solution looks like https://github.com/mate-academy/jv-fruit-shop

Reply:
please reply to this thread